src/payment/cart/CouponForm.jsx

```diff
@@ -35,7 +35,7 @@ class CouponForm extends Component {
 
   handleRemoveSubmit(event) {
     event.preventDefault();
-    this.props.removeCoupon({ code: this.props.id });
+    this.props.removeCoupon({ id: this.props.id });
```

Comment:
This was misleading - this is the voucher ID here, not the coupon code.  Updated it to remove ambiguity.